src/keys/seed.rs

```diff
@@ -44,7 +45,7 @@ impl Seed {
 }
 
 impl FromStr for Seed {
-    type Err = anyhow::Error;
+    type Err = errors::FeelessError;
```

Comment:
I reckon we always import FeelessError directly, rather than the error module. It's obvious enough who owns the type.